### PR TITLE
fix some errors

### PR DESCRIPTION
**Pull request by lixinyilalala**
fix some tiny problems

### CODE REVIEW

File: src/test/java/com/hubspot/dropwizard/guice/objects/TestModule.java [hunk 1]
```diff
@@ -6,7 +6,6 @@
 public class TestModule extends AbstractModule {
     @Override
     protected void configure() {
-        bind(ExplicitDAO.class);
```
Comment (jhaber) [inline src/test/java/com/hubspot/dropwizard/guice/objects/TestModule.java]: Why remove this binding?

**Comment by lixinyilalala** [comment]
image

**Comment by jhaber** [comment]
Judging by the name and the contents the explicit binding is intentional. Since the HK2 interactions are complicated it seems reasonable to test just-in-time bindings and explicit bindings separately since they are fundamentally different.

**Comment by lixinyilalala** [comment]
You are right, maybe I should close this pull request, and push a new commit to fix another two problems~

**Pull request closed without merging**